vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -1378,6 +1376,8 @@ public Grid(Class<T> beanType) {
         addDragEndListener(this::onDragEnd);
 
         getElement().setAttribute("suppress-template-warning", true);
+
+        getElement().setProperty("__deselectAllowed", true);
```

Comment:
Should we rename the property to `__deselectDisallowed` so we wouldn't need to set a truthy default value for it?

Reply:
Done.